questions.md

```diff
@@ -0,0 +1 @@
+What questions DO I have about pull requests?
```

Comment:
@e3-cerruti I think you are pretty awesome at PRs, but here's a primer on merge conflicts in the classroom if you need one :) ✨ https://github.blog/2018-08-22-merge-conflicts-in-the-classroom/